assets/js/base/components/cart-checkout/order-meta/index.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+import { createSlotFill } from 'wordpress-components';
```

Comment:
Now that plugin area is merged, we can test this.
I suspect it won't work because of this custom `wordpress-components` therefore we end up with a different context.
I will try to do some testing.

Reply:
So I tested this and I was right, it won't work unless we export `SlotFillProvider` from our repo.